conf/config.properties

```diff
@@ -31,10 +31,12 @@ smtp.host.encryption=tls
 smtp.host.port=465
 
 # SMTP sender displayname (the name displayed to the recipient), email, username (to login) and password. Usually email and username are the same (but not necessarily)
-smtp.sender.displayname=Admin
```

Comment:
@Orbiter Need suggestion, if these config key name should be changed.

Reply:
Yes, the display name of the sender should be an option, that can be defined in the frontend.